integration_tests/controllers_test/drcluster_mmode_test.go

```diff
@@ -57,8 +57,8 @@ var _ = Describe("DRClusterMModeTests", Ordered, func() {
 
 		testEnv = &envtest.Environment{
 			CRDDirectoryPaths: []string{
-				filepath.Join("..", "config", "crd", "bases"),
-				filepath.Join("..", "hack", "test"),
+				filepath.Join("..", "..", "config", "crd", "bases"),
+				filepath.Join("..", "..", "hack", "test"),
```

Comment:
I'm not sure we need to separate the integration tests, but looking at the makefile changes I like that we avoid the testenv work for the unit tests. However adding another level of nested directories seems like the direction.

Why not move the tests from controllers to inegration_tests dropping the controller_test directory?